internal/controller/supervisorstorage/garbage_collector_test.go

```diff
@@ -121,7 +121,8 @@ func TestGarbageCollectorControllerSync(t *testing.T) {
 
 	spec.Run(t, "Sync", func(t *testing.T, when spec.G, it spec.S) {
 		const (
-			installedInNamespace = "some-namespace"
+			installedInNamespace         = "some-namespace"
+			currentSessionStorageVersion = "7" // update this when you update the storage version in the production code
```

Comment:
Will some test fail if there's a mismatch between this version and the production code version?

Reply:
Yes, all the tests within that file which use that variable should fail when the production code has a different value.